### ISSUE TITLE
[aria-label] console reports aria-label not set when string is inside ng-if

### ISSUE THREAD

**Issue by mpiasta-ca** [angular/material#8494]
**Actual behavior**:
-  What is the issue? aria-label is not set using element contents, as expected, because the contents are inside an `ng-if="..."` element (where `ng-show="..."` works w/o issues). Console reports error of aria-label not being set.
-  What is the expected behavior? Should be able to set the aria-label using the string of the element post-render.

**CodePen** or Steps to reproduce the issue: *
-  CodePen Demo which shows your issue: http://codepen.io/anon/pen/aNrgyV?editors=1010
-  Details: the md-button has a dynamic label (it's a submit button that either says "Save Changes" or "Create" based on whether the page is used as Edit or as Create New Record). The aria-label is not set because it cannot deduce on its own which string to use for the aria-label.

**Angular Versions**: *
-  Angular Version: 1.5
-  Angular Material Version: 1.0.8...? (used codepen template)

**Additional Information**:
-  Browser Type: *all
-  Browser Version: *all
-  OS: *all
-  Stack Traces: n/a

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our ‘Surge Focus on Material 2' efforts.
      For details, see our forum posting @ http://bit.ly/1UhZyWs.
